opencti-platform/opencti-graphql/src/database/engine.js

```diff
@@ -2990,6 +2990,47 @@ export const elAggregationRelationsCount = async (context, user, indexName, opti
     });
 };
 
+export const elAggregationNestedTermsWithFilter = async (context, user, indexName, aggregation, opts = {}) => {
+  const { types = [], size = 500 } = opts;
```

Comment:
Could we use one size parameter for the default value here ? like default_pagination_result for example ?